src/pages/UpdateServers.js

```diff
@@ -447,9 +509,11 @@ class UpdateServers extends BaseUpdateWizardPage {
   }
 
   render() {
+    let loadingText =  this.state.validating ? translate('server.validating') : '';
     return (
       <div className='wizard-page'>
-        <LoadingMask show={this.state.wizardLoading || this.state.loading}/>
+        <LoadingMask show={this.state.wizardLoading || this.state.loading || this.state.validating}
```

Comment:
wizardLoading should not be in state. It should just use this.props.wizardLoading. The same is true for wizardLoadingErrors. This will also require removing `compnentDidUpdate` (and moving some of that logic into the constructor).

Reply:
will reuse existing loading flag. Used it with undefined, '' and 'Validating Changes' to indicate no loading, loading and validating. 
will update compnentDidUpdate